src/hiccup/compiler.clj

```diff
@@ -314,22 +333,24 @@
 (defmethod compile-element ::literal-tag
   [[tag attrs & content]]
   (let [[tag tag-attrs _] (normalize-element-form [tag])
-        attrs-sym         (gensym "attrs")]
+        attrs-sym (gensym "attrs")]
     `(let [~attrs-sym ~attrs]
-       (if (map? ~attrs-sym)
-         ~(if (container-tag? tag content)
-            `(build-string ~(str "<" tag)
-                           (render-attr-map (merge ~tag-attrs ~attrs-sym)) ">"
-                           ~@(compile-seq content)
-                           ~(str "</" tag ">"))
-            `(build-string ~(str "<" tag)
-                           (render-attr-map (merge ~tag-attrs ~attrs-sym))
-                           ~(end-tag)))
-         ~(if (container-tag? tag attrs)
-            `(build-string ~(str "<" tag (render-attr-map tag-attrs) ">")
-                           ~@(compile-seq (cons attrs-sym content))
-                           ~(str "</" tag ">"))
-            (build-string "<" tag (render-attr-map tag-attrs) (end-tag)))))))
+       (build-string
```

Comment:
Ah, I see. I'd missed that the `if` and `when` forms are concatenated.

Could you change the indentation so that it's:

```clojure
(build-string
 (if (map? ~attrs-sym)
   ...)
 (when ...))
```

So indented one space, rather than two. Hiccup follows the Clojure Style Guide for it's indentation, and for whatever reason Cursive differs from it.

```clojure
;; Style guide on how to indent lists by default
(foo
 bar)
;; Cursive on how to indent lists by default
(foo
  bar)
```